src/widgets/MeterPanel.cpp

```diff
@@ -341,6 +342,17 @@ MeterPanel::MeterPanel(AudacityProject *project,
    mRuler.SetLabelEdges(true);
    //mRuler.SetTickColour( wxColour( 0,0,255 ) );
 
+   mSlider = std::make_unique<LWSlider>(this, XO(""),
```

Comment:
There are MeterPanels also in the Mixer board, which you can open using the View menu.  It makes a control for each audio track, and a meter panel with isInput = false.

Did you test this too?

I don't see unintended changes in the Mixer board yet resulting from this change.  No slider is drawn over that meter.  I don't understand why not, but it is as it should be.